components/net_traits/lib.rs

```diff
@@ -459,6 +459,7 @@ pub enum ResourceAttribute {
     RedirectCount(u16),
     RequestStart,
     ResponseStart,
+    RedirectStart,
```

Comment:
Let's add this:

```rust
enum RedirectStartValue {
    Zero,
    FetchStart,
}
```
And then modify the ResourceAttribute enum to have `RedirectStart(RedirectStartValue)`.